### PR TITLE
Added Inconsistencies LaTeX file under Resources

### PR DESCRIPTION
**Pull request by EmilSoleymani**
Still lots more to add

### CODE REVIEW

File: Resources/Inconsistencies.tex [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+\newcommand{\eli}{\end{itemize}}
+
+\begin{document}
+
+    %% Title Page %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+    \begin{center}
+
+        {\large \textbf{VDispl}}\\[2mm]
+        {\large \textbf{Soil Settlement Analysis Software}}\\[2mm]
+        {\huge \textbf{Inconsistencies in Settlements Textbook and FORTRAN Code}}\\[6mm]
```
Comment (smiths) [inline Resources/Inconsistencies.tex:14]: The `\large` and `\huge` text throws off the title spacing.  You don't need them.  (If you really want the larger text, you could look to see if there is a LaTeX class file with the title display that you like.)

File: Resources/Inconsistencies.tex [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        {\large \textbf{Dr.~Spencer Smith, Emil Soleymani}}\\[2mm]
```
Comment (smiths) [inline Resources/Inconsistencies.tex:15]: You should list yourself as first author Emil.

File: Resources/Inconsistencies.tex [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+        {\large Revised: May 9, 2022}
+      
+      \end{center}
+      
+      \medskip
+
+      % Description
+      \indentpar During review of the legacy FORTRAN code for the VDispl software, 
+      we referenced the Settlements Analysis book linked in the README 
+      file in order to correlate each calculation with its corresponding
```
Comment (smiths) [inline Resources/Inconsistencies.tex:26]: Whenever you write "in order to" you can probably replace it with "to".  This is an example of a low information content phrase.  You can find examples of things to watch for like this in my [writing checklist](https://gitlab.cas.mcmaster.ca/smiths/cas741/-/blob/master/BlankProjectTemplate/docs/Writing-Checklist.pdf).

**Review by smiths — CHANGES_REQUESTED** [review]
I've requested some changes with my annotations to the commit on GitHub.

**Comment by smiths** [comment]
I have another requested change for this PR.  The Resources folder is not the right place.  The Resources folder is for external resources to our project, like Dr. Stolle's files.  I have another meeting to get to, but then I'll get back to this with a suggestion on where we should put it.  :smile:

**Comment by EmilSoleymani** [comment]
First 3 requested changes are made. Waiting on suggestion for where to move .tex and .pdf file before pushing changes.

**Comment by smiths** [comment]
For now, create a folder with the name Miscellaneous.  Once we see what else goes in there, we can probably come up with a more meaningful name.  :smile:

File: Resources/Inconsistencies.tex (hunk 1 above)
Comment (EmilSoleymani) [reply Resources/Inconsistencies.tex]: Found new Title Page template

File: Resources/Inconsistencies.tex (hunk 3 above)
Comment (EmilSoleymani) [reply Resources/Inconsistencies.tex]: Sentence changed

**Review by smiths — APPROVED** [review]
Looks great!

**Pull request merged**